charts/dsp-atlantis/values.yaml

```
@@ -13,9 +13,39 @@ atlantis:
   fullnameOverride: atlantis
   serviceAccount:
     name: atlantis
+
 azure:
   enabled: false
   certificateFile: "azure-cert.pfx"
 vault:
   azure:
     pathPrefix: /path/to/azure/secrets
```

```suggestion
vault:
  azure:
    pathPrefix: /path/to/azure/secrets
  cleanup:
    ...
```
Can you just add the cleanup credentials in this vault block rather than making a separate one.

---

Ah yep, I think this stemmed from the merge conflict I hurriedly fixed yesterday morning--will do